docs/source/release/v3.9.0/framework.rst

```diff
@@ -108,8 +102,8 @@ Improved
 Python
 ------
 
-- The function `IMDDimension.getName()` has been deprecated. Use the propery `IMDDimension.name` instead.
-- The duplicate function `Workspace.getName()` has been deprecated. Use `Workspace.name()` instead.
+- The function `IMDDimension.getName()` has been deprecated in favour of `IMDDimension.name'.
+- The duplicate function `Workspace.getName()` has been deprecated in favour of `Workspace.name()'.
 
 Python Algorithms
```

Comment:
Can you get rid of the python algorithm and bug fixes sections and merge them into the new and improved algorithms sections